tests/framework/cli/test_cli.py

```diff
@@ -71,6 +77,16 @@ def fake_session(mocker):
     return mocked_session
 
 
+class TestDefaultLogging:
+    def test_setup_root_logger(self):
+        root_logger = logging.getLogger()
+        assert "console" in {handler.name for handler in root_logger.handlers}
+
+    def test_setup_kedro_logger(self):
+        kedro_logger = logging.getLogger("kedro")
+        assert kedro_logger.level == logging.INFO
```

Comment:
These are updated versions of tests that used to be in `test_cli_logging_setup.py` (tests were in the wrong place before, now that file is deleted).